src/vs/workbench/contrib/tasks/common/taskTemplates.ts

```diff
@@ -29,6 +29,11 @@ const dotnetBuild: TaskEntry = {
 		'\t\t\t"label": "build",',
 		'\t\t\t"command": "dotnet build",',
 		'\t\t\t"type": "shell",',
+		'\t\t\t"args": [',
+		'\t\t\t\t// Ask dotnet build to generate full paths for file names.',
+		'\t\t\t\t"/p:GenerateFullPaths=true",',
```

Comment:
nit: Consider using the same longer /property:GenerateFullPaths=true spelling as in the msbuild case below.

Reply:
Done. Thank you